app/routes.rb

```diff
@@ -144,3 +145,8 @@ def notify_upsert_exam(exam_id)
 require_relative './routes/notifications'
 require_relative './routes/suggestions'
 
+options '*' do
+  response.headers['Allow'] = settings.allow_methods.map { |it| it.to_s.upcase }.join(',')
```

Comment:
Why do we need cross origin support? Isn't this app supposed to already implement it?

Reply:
Yes I need it. I don't know why it fails, but adding the option endpoint doen't fail any more.